Makefile

```
@@ -91,7 +91,11 @@ templates-generate:
 	@hack/templates.sh
 
 .PHONY: generate-all
-generate-all: generate manifests templates-generate add-license
+generate-all: generate manifests templates-generate add-license projectsveltos-crds
+
+.PHONY: projectsveltos-crds
+projectsveltos-crds: sveltos-crds
```

I think it can be simplified
```suggestion
projectsveltos-crds: $(SVELTOS_CRD)
```
and the line 524 can be removed.

---

The thing is it does not work as you suggested. Maybe local mac issue, but I didn't manage to run target in format `rojectsveltos-crds: $(SVELTOS_CRD)`. Thus I decided to keep as in PR, bc in that format it will work.